platform/osgi/platform-osgi-conditions/src/main/java/org/codice/ddf/condition/BundleNameCondition.java

```diff
@@ -27,7 +27,7 @@
  *
  * <p>Selects bundles based on the name. @ThreadSafe
  */
-public class BundleNameCondition implements Condition {
+public class BundleNameCondition extends AbstractCondition implements Condition {
```

Comment:
The first paragraph is the same, there is a description under that which tells what the specific implementation does.

Reply:
Just wondering why we keep repeating the same thing in all of them. Especially since they are _not_ interfaces but classes. We could just remove the first part and keep the part that explains what the implementation does.

Reply:
I dunno, that is just how I did it :)